papis/web/tags.py

```diff
@@ -35,17 +35,26 @@ def tags_list_div(tags: Tags, libname: str) -> None:
             _tag(tag=tag, libname=libname)
 
 
-def html(pretitle: str, libname: str, tags: Dict[str, int]) -> t.html_tag:
+def html(pretitle: str, libname: str, tags: Dict[str, int],
+         sort_by: str) -> t.html_tag:
     with papis.web.header.main_html_document(pretitle) as result:
         with result.body:
             papis.web.navbar.navbar(libname=libname)
             with wh.container():
                 with t.h1("TAGS"):
                     with t.a(href=f"/library/{libname}/tags/refresh"):
                         wh.icon("refresh")
+                    with t.a(href=f"/library/{libname}/tags?sort=alpha",
+                             title="sort by name"):
+                        wh.icon("arrow-down-a-z")
+                    with t.a(href=f"/library/{libname}/tags?sort=numeric",
+                             title="sort by number of occurences"):
+                        wh.icon("arrow-down-1-9")
                 with wh.container():
-                    for tag in sorted(tags,
-                                      key=lambda k: tags[k],
-                                      reverse=True):
+                    # either sort by number of occurences or alphabetical
+                    sort_kwargs = dict(key=lambda k: tags[k], reverse=True)
+                    if sort_by == "alpha":
+                        sort_kwargs = dict()
+                    for tag in sorted(tags, **sort_kwargs):
```

Comment:
```suggestion
                    # either sort by number of occurrences or alphabetical
                    if sort_by == "alpha":
                    	tags = sorted(tags)
                    elif sort_by in ("", "numeric"):
                    	tags = sorted(tags, key=lambda k: tags[k], reverse=True)
                    for tag in tags:
```
? In case we ever want to add more sorting formats?